<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>29</y>
     <width>1071</width>
     <height>791</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="login_section">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>291</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>0</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="order_section">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>291</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inquire_section">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>140</y>
       <width>291</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string>문의</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
